### PR TITLE
made xavier parameter right in train_model.py

### PR DESCRIPTION
**Pull request by realbns2008**
as title

### CODE REVIEW

**Comment by winstywang** [comment]
Why do we need this? Xavier or the MSRA is the best for initialization, though I think the current parameter for Xavier is not correct.

**Comment by realbns2008** [comment]
@winstywang the issue is #1422
if use the previous default initializer the alexnet and googlenet didn't convergence.
Although by using normal initializer they do convergence

**Comment by winstywang** [comment]
The reason is the parameter of Xavier is not correct. Try use mx.init.Xavier(factor_type="in", rnd_type="gaussian", magnitude=2.0) for MSRA initialization. It works well for the network with BN. For alexnet, you need to divide the output by 128 in the FC layer.

**Comment by javelinjs** [comment]
@winstywang I think, despite of providing the model definition, examples should also provide proper initializers. Did you mean that Xavier can fit all those models, with the same parameters?
@realbns2008 But I don't think the modification you made is good enough. Changing train.py doesn't organize things well.

**Comment by winstywang** [comment]
@javelinjs Yes, you could refer to Kaiming's paper on analysis on initialization for deep neural network.

**Comment by javelinjs** [comment]
@winstywang I see, thanks. Seems that we should change the params for Xavier.

**Comment by realbns2008** [comment]
@winstywang I use the mx.init.Xavier(factor_type="in", rnd_type="gaussian", magnitude=2.0) for example model googlenet , but it diden't converge. However alex net did converge after using the right Xavier.  But what about googlenet, I've tried the mx.init.Normal(), but it did'nt work as well.

**Comment by winstywang** [comment]
@realbns2008 Is your googlenet with BN?

File: example/image-classification/train_model.py [hunk 1]
```diff
@@ -73,8 +73,8 @@ def fit(args, network, data_loader, sym_gen=None):
         learning_rate      = args.lr,
         momentum           = 0.9,
         wd                 = 0.00001,
-        initializer        = mx.init.Xavier(factor_type="in", magnitude=2.34),
         sym_gen            = sym_gen,
+        initializer        = mx.init.Xavier(actor_type="in", rnd_type="gaussian", magnitude=2.0),
```
Comment (jmschrei) [inline example/image-classification/train_model.py]: I think you mean factor type.

**Comment by realbns2008** [comment]
@winstywang  no , not with BN.  BN seems not care about how you initialize the weight.

File: example/image-classification/train_model.py (hunk 1 above)
Comment (realbns2008) [inline example/image-classification/train_model.py]: @jmschrei  no i mean the rand_type, default is uniform distribution, which may not be right. 
according to this paper http://arxiv.org/pdf/1502.01852v1.pdf  when the active type is relu , we should use mx.init.Xavier(actor_type="in", rnd_type="gaussian", magnitude=2.0)

Comment (realbns2008) [inline example/image-classification/train_model.py]: @jmschrei sorry i made a poor spelling mistake, thanks for notice it :)

**Comment by realbns2008** [comment]
@winstywang  it seems my PR triggered some travis test error, could you help me rerun the test ?

**Comment by antinucleon** [comment]
Close for long time idle

**Pull request closed without merging**